app/components/primer/beta/auto_complete.rb

```diff
@@ -32,14 +32,41 @@ class AutoComplete < Primer::Component
         Primer::BaseComponent.new(**system_arguments)
       }
 
+      # Customizable input used to search for results.
+      # It is preferred to use this slot sparingly - it will be created by default if not explicity added.
```

Comment:
In what scenarios could it make sense to use this `input` slot? Would it be when someone wants to set custom `system_arguments` on `input`?

Reply:
Yes, we have examples where this is necessary in dotcom (for example when custom classes are applied).